index.bs

```diff
@@ -4478,33 +4475,22 @@ entails in the ECMAScript language binding.
 <div class="example">
 
     Here is are some examples of wording to use to create and throw exceptions.
-    To throw a new [=simple exception=] named
-    {{TypeError}}:
+    To throw a new [=simple exception=] named <emu-val>TypeError</emu-val>:
 
     <blockquote>
-
-        [=Throw=] a TypeError.
-
+        [=Throw=] a <emu-val>TypeError</emu-val>.
```

Comment:
So in that case, we need to define each simple exception somewhere (so back out of the changes right below [#dfn-simple-exception](https://cdn.rawgit.com/tobie/webidl/ab2d126/index.html#dfn-simple-exception)).